Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLinkMidiControl.cs

```diff
@@ -0,0 +1,337 @@
+﻿
+using UdonSharp;
+using UnityEngine;
+using VRC.SDKBase;
+using VRC.Udon;
+
+namespace AudioLink
+{
+
+    [UdonBehaviourSyncMode(BehaviourSyncMode.None)]
+    public class AudioLinkMidiControl : UdonSharpBehaviour
+    {
+        public AudioLinkController _AudioLinkController;
+        public bool allowMidi;
```

Comment:
what's the point of having a boolean to disable the behavior? You can just disable the component itself, this seems redundant to me.

Reply:
Setting the component's .enabled state doesn't prevent the "VRC Midi Listener" from calling it's functions, so a allowMidi bool is required to allow for toggling it.